app/models/sealion.rb

```diff
@@ -0,0 +1,5 @@
+class Sealion < ActiveRecord::Base
+  # Remember to create a migration!
+  validates :first_name, {:presence => true}
```

Comment:
I'm curious about the tradeoffs of doing this vs. specifying null: false in the migration. It seems like it would be preferable to specify this in the migration and have the database verify that first and last name can never be null, but I'm not sure on the tradeoffs—thoughts?